Android/res/values-es/arrays.xml

```diff
@@ -0,0 +1,270 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Ups! in this resource I want only add the lines
 <item>15 seg.</item>
<item>30 seg.</item>
<item>45 seg.</item>
<item>60 seg.</item>

in spanish segundos (seg)

I forgot to delete the rest